Packages/Sites/Sfi.Kateheo/Configuration/NodeTypes.yaml

```diff
@@ -125,7 +125,7 @@ blah: &alohaRich
       validation:
         'TYPO3.Neos/Validation/DateTimeValidator': []
     author:
```

Comment:
Makes sense to rename to `authors` and add the migration from to move data from `author` to `authors`